runtime/jvmti/jvmtiStartup.c

```diff
@@ -519,15 +518,13 @@ loadAgentLibraryGeneric(J9JavaVM * vm, J9JVMTIAgentLibrary * agentLibrary, char
 	rc = issueAgentOnLoadAttach(vm, agentLibrary, agentLibrary->options, loadFunctionName, found);
 	if (!(*found)) {
 		/* If the load function wasn't found, issueAgentOnLoadAttach already closed the library. */
-		Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed1(loadFunctionName);
-		Trc_JVMTI_loadAgentLibraryGeneric_Exit(agentLibrary->nativeLib.name);
+		Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed1_Exit(agentLibrary->nativeLib.name, loadFunctionName);
 		return rc;
 	}
 
 	/* For errors other than load function not found ... */
 	if (JNI_OK != rc) {
-		Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed2(rc);
-		Trc_JVMTI_loadAgentLibraryGeneric_Exit(agentLibrary->nativeLib.name);
+		Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed2_Exit(agentLibrary->nativeLib.name, loadFunctionName, rc);
```

Comment:
Please add an initial value for `rc`:
```c
	jint rc = JNI_OK;
```

Reply:
Added, also refactored `loadAgentLibraryGeneric()`.